telemetry/roles/grafana/tasks/plugins.yml

```diff
@@ -20,8 +20,14 @@
     version: "{{ grafana_plugins_stable_commit }}"
 
 - name: Wait for grafana pod to come to ready state
-  ansible.builtin.command: kubectl wait --for=condition=ready --timeout="{{ grafana_pod_timeout }}" -n "{{ grafana_namespace }}" pod -l app="{{ grafana_k8s }}"
-  changed_when: false
+  block:    
+    - name: Wait for grafana pod to come to ready state
+      ansible.builtin.command: kubectl wait --for=condition=ready --timeout="{{ grafana_pod_timeout }}" -n "{{ grafana_namespace }}" pod -l app="{{ grafana_k8s }}"
```

Comment:
Why don't you use Ansible K8s collections: https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_module.html

Reply:
it was failing due to below requirement. Currently we dont install python on each compute node.
![image](https://github.com/user-attachments/assets/7691495b-fa2c-4083-8db0-993b1240e1d6)

Reply:
You should use K8s and Helm collections from the workstation node where you launch the playbook. This shouldn't be done in each of the control or compute nodes.
Using these collections, they consume K8s APIs and therefore you don't need to be on the cluster nodes to consume it.

Reply:
As discussed, omnia will support using k8s collections instead of commands in next release